Func.Isolated.Net7.With.AI/Func.Isolated.Net7.With.AI.csproj

```diff
@@ -15,10 +15,10 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.10.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.11.0-preview2" />
```

Comment:
Any reason for the 2 preview packages?

Reply:
i got it just working with the preview packages, set them all to preview and then it worked